Sources/Application/SceneLifecycle.swift

```diff
@@ -19,7 +19,7 @@ extension SceneLifecycle {
     
     func promptToUnlock() {
         coordinator?.evaluateRevisit {
-            windowManager?.hideUnlockWindow()
+            self.windowManager?.hideUnlockWindow()
```

Comment:
Is this needed?

Reply:
Yes, because this is now an escaping closure.  Compiler complains otherwise

Reply:
Worth capturing self as unowned?
```swift
func promptToUnlock() {
    coordinator?.evaluateRevisit { [unowned self] in
        windowManager?.hideUnlockWindow()
    }
}
```
?